pkg/cmd/recoveryapiserver/create.go

```diff
@@ -19,16 +18,12 @@ type CreateOptions struct {
 	Options
 
 	StaticPodResourcesDir string
-	Timeout               time.Duration
-	Wait                  bool
```

Comment:
@deads2k I think you should have kept at least the struct fileds or unit testing becomes real pain